arch/arm64/configs/bcm2711_defconfig

```diff
@@ -1031,6 +1031,7 @@ CONFIG_USB_G_HID=m
 CONFIG_USB_G_WEBCAM=m
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
+CONFIG_MMC_SDHCI_BCM2711=y
```

Comment:
This is a vestigial setting and should be removed.

Reply:
The commit is to revert 9d1deec93f "configs: arm64/bcm2711: Remove CONFIG_VIDEO_BCM2835".
If that commit contained other stuff than removing CONFIG_VIDEO_BCM2835 then it shouldn't have been merged as it stood, and should have been taken up with the author of that commit.

Re https://github.com/raspberrypi/linux/commit/9d1deec93fa8b1b4953ff5e9210349f3c85b9a8d
> I always save_defconfig, so the other changes aren't really changes at all.

Nor are whitespace changes really changes, but those aren't acceptable in patches that are also doing something else, nor without the commit text saying that it is doing that.
"Because my tool did it" means that you've got 2 commits - the cleanup so your tool is happy, then the change.

Sorry, different rules for different people annoys me.

I've created a new commit to remove it.